integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/ActionConstants.java

```diff
@@ -50,8 +50,8 @@ public interface ActionConstants {
       "https://github.com/oracle/weblogic-deploy-tooling/releases/latest";
   public static final String WDT_DOWNLOAD_URL =
       getNonEmptySystemProperty("wko.it.wdt.download.url", WDT_DOWNLOAD_URL_DEFAULT);
-  public static final String WDT_VERSION =
-      getNonEmptySystemProperty("wko.it.wdt.version", "latest");
+  public static final String WDT_VERSION = "latest";
```

Comment:
So we assume that WDT_VERSION string is always latest if we download the zip from main branch or any released branch such as wdt version 2.2.1

Reply:
Yes. Here WDT_VERSON is used as an identifier that tells you under which directory you want to put weblogic-deploy.zip file so that later on  n the test class when Mii domain is being built WIT knows from where to find weblogic-deploy.zip and add it to its cache.